python/letsql/common/caching.py

```diff
@@ -105,7 +105,7 @@ class ParquetCacheStorage(CacheStorage):
     path = field(
         validator=instance_of(pathlib.Path),
         converter=abs_path_converter,
-        default=(pathlib.Path("~/.local/share/letsql")),
+        default=letsql.options.cache_default_path,
```

Comment:
rather than `default`, this `field` should use `factory` to dynamically get the `cache_default_path` value
something like `factory=functools.partial(letsql.options.get, "cache_default_path")` should work

https://www.attrs.org/en/stable/examples.html#defaults